### PR TITLE
[Workers]: Add Upstash Database Integration Docs

### PR DESCRIPTION
**Pull request by lauragift21**
This PR adds documentation for Upstash integration with Cloudflare Workers

### CODE REVIEW

File: content/workers/learning/integrations/databases.md [hunk 1]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
 
 To learn more about Neon, refer to [Neon's official documentation](https://neon.tech/docs/introduction).
 
+### Upstash
+
+[Upstash](https://upstash.com/) is a serverless database with Redis API. It aims to simplify the process of building and scaling real-time applications by providing a serverless database infrastructure.
+
+1. To set up an integration with Upstash, you to have an existing Upstash database to connect to. [Create a Upstash database](https://docs.upstash.com/redis#create-a-database) or [load data from an existing database to Upstash](https://docs.upstash.com/redis/howto/connectclient).
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
1. To set up an integration with Upstash, you need an existing Upstash database to connect to. [Create a Upstash database](https://docs.upstash.com/redis#create-a-database) or [load data from an existing database to Upstash](https://docs.upstash.com/redis/howto/connectclient).
```

File: content/workers/learning/integrations/databases.md [hunk 2]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
+
+2. Insert some data to your Upstash database. You can add data to your Upstash database in two ways:
+    - Use the CLI directly from your Upstash console
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
    - Use the CLI directly from your Upstash console.
```

File: content/workers/learning/integrations/databases.md [hunk 3]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
+3. Add the Upstash database integration to your Worker:
+    - Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+    - In **Account Home**, select **Workers & Pages**.
+    - In **Overview**, select your Worker.
+    - Select **Settings** > **Integrations** > **Upstash**. 
+    - Follow the setup flow, selecting the database created in step 1.
+
+4. In your Worker, install the `@upstash/redis`, a HTTP client to connect to your database and start manipulating data:
+
+    ```
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
    ```sh
```

File: content/workers/learning/integrations/databases.md [hunk 4]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
+    npm install @upstash/redis
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
    $ npm install @upstash/redis
```

File: content/workers/learning/integrations/databases.md [hunk 5]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
+          return new Response(greeting);
+        }
+      }
+
+      return new Response("Hello!");
+    },
+  };
+  ```
+
+To learn more about Upstash, refer to [Upstash's official documentation](https://docs.upstash.com/redis).
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
To learn more about Upstash, refer to the [Upstash documentation](https://docs.upstash.com/redis).
```

**Review by deadlypants1973 — COMMENTED** [review]
PCX

File: content/workers/learning/integrations/databases.md [hunk 6]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
 
+### Upstash
+
+[Upstash](https://upstash.com/) is a serverless database with Redis API. It aims to simplify the process of building and scaling real-time applications by providing a serverless database infrastructure.
+
+1. To set up an integration with Upstash, you to have an existing Upstash database to connect to. [Create a Upstash database](https://docs.upstash.com/redis#create-a-database) or [load data from an existing database to Upstash](https://docs.upstash.com/redis/howto/connectclient).
+
+2. Insert some data to your Upstash database. You can add data to your Upstash database in two ways:
+    - Use the CLI directly from your Upstash console
+    - Alternatively, install `redis-cli` locally and run the following commands.
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]: Can we link to the `redis-cli` installation guide here?

Comment (lauragift21) [reply content/workers/learning/integrations/databases.md]: Yes sure! I'll include the link.

File: content/workers/learning/integrations/databases.md [hunk 7]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
 
 To learn more about Neon, refer to [Neon's official documentation](https://neon.tech/docs/introduction).
 
+### Upstash
+
+[Upstash](https://upstash.com/) is a serverless database with Redis API. It aims to simplify the process of building and scaling real-time applications by providing a serverless database infrastructure.
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
[Upstash](https://upstash.com/) is a serverless database with Redis and Kafka support.
```

File: content/workers/learning/integrations/databases.md [hunk 8]
```diff
@@ -254,6 +255,62 @@ To learn more about Supabase, refer to [Supabase's official documentation](https
...
+
+1. To set up an integration with Upstash, you need an existing Upstash database to connect to. [Create a Upstash database](https://docs.upstash.com/redis#create-a-database) or [load data from an existing database to Upstash](https://docs.upstash.com/redis/howto/connectclient).
```
Comment (deadlypants1973) [inline content/workers/learning/integrations/databases.md]:
```suggestion
To set up an integration with Upstash:

1. You need an existing Upstash database to connect to. [Create a Upstash database](https://docs.upstash.com/redis#create-a-database) or [load data from an existing database to Upstash](https://docs.upstash.com/redis/howto/connectclient).
```

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Pull request closed without merging**